### PR TITLE
CRS.Simple tutorial

### PR DESCRIPTION
**Pull request by IvanSanchez**
Implements a bit of #4239. Also: spaceships!

### CODE REVIEW

File: examples/crs-simple/crs-simple.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+The game has a built-in square coordinate system, as can be seen in the corners. This will allow us to establish a coordinate system.
+
+<center>
+<img src="uqm_map_detail.png" style="border: 1px solid #ccc; border-radius: 5px" /><br/>
+</center>
+
+
+## CRS.Simple
+
+**CRS** stands for [coordinate reference system](https://en.wikipedia.org/wiki/Spatial_reference_system), a term used by geographers to explain what the coordinates mean in a coordinate vector. For example, `[15, 60]` represents a point in the indic ocean if using latitude-longitude on the earth, or the solar sistem Krueger-Z in our starmap.
```
Comment (yohanboniface) [inline examples/crs-simple/crs-simple.md]: indic? Indian? :)

**Comment by yohanboniface** [comment]
You roca! ;)

**Comment by IvanSanchez** [comment]
I forgot to send kudos to @AlanDeSmet for the Ur-Quan Masters maps, though :-)

File: examples/crs-simple/crs-simple-example3.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		var deneb    = xy(218.7,   8.3);
+
+		L.marker(     sol).addTo(map).bindPopup(      'Sol');
+		L.marker(   mizar).addTo(map).bindPopup(    'Mizar');
+		L.marker(kruegerZ).addTo(map).bindPopup('Krueger-Z');
+		L.marker(   deneb).addTo(map).bindPopup(    'Deneb');
+
+		var travel = L.polyline([sol, deneb]).addTo(map);
+
+		map.setView( xy(120, 70), 1);
```
Comment (jieter) [inline examples/crs-simple/crs-simple-example3.html]: Space before `xy` intentional?

File: examples/crs-simple/crs-simple-example3.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+	<script type="text/javascript" src="eu-countries.js"></script>
+
+	<script>
+
+		var map = L.map('map', {
+			crs: L.CRS.Simple,
+			minZoom: -3
+		});
+
+		var YX = L.LatLng;
```
Comment (jieter) [inline examples/crs-simple/crs-simple-example3.html]: You do not use `YX` here. Might just want to remove it?

File: examples/crs-simple/crs-simple-example3.html (hunk 2 above)
Comment (IvanSanchez) [inline examples/crs-simple/crs-simple-example3.html]: I like my code to line up vertically :-)

File: examples/crs-simple/crs-simple-example3.html (hunk 3 above)
Comment (IvanSanchez) [inline examples/crs-simple/crs-simple-example3.html]: Yeah, why not.

File: examples/crs-simple/crs-simple-example3.html (hunk 2 above)
Comment (jieter) [inline examples/crs-simple/crs-simple-example3.html]: I see, but did not see where it aligns to in this case. Wasn't commenting on the other space-before instances...

Comment (IvanSanchez) [inline examples/crs-simple/crs-simple-example3.html]: Ah, yeah, that's the part of my brain which completely ignores whether I should have a space just after a parenthesis or not :-)

Comment (jieter) [inline examples/crs-simple/crs-simple-example3.html]: Lucky you

File: examples/crs-simple/crs-simple.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+	L.marker(sol).addTo(map);
+	map.setView( [70, 120], 1);
+
+<iframe src='crs-simple-example2.html' width='616' height='416'></iframe>
+
+You can see [this example](crs-simple-example2.html) stand-alone.
+
+### This is not the `LatLng` you're looking for
+
+You'll notice that Sol is at coordinates `[145,175]` instead of `[175,145]`, and the same happens with the map center. Coordinates in `CRS.Simple` take the form of `[y, x]` instead of `[x, y]`, in the same way Leaflet uses `[lat, lng]` instead of `[lng, lat]`.
```
Comment (jieter) [inline examples/crs-simple/crs-simple.md]: more space-nitpicking: space with and without comma.

**Pull request merged**